configs/avae-test-config.yml

```diff
@@ -0,0 +1,51 @@
+#### Data parameters
+
+datapath: data/shrec21_full_dataset/subtomo_files
+affinity: data/shrec21_full_dataset/subtomo_files/affinity_scores.csv
+classes: data/shrec21_full_dataset/subtomo_files/classes.csv
+limit: 1000
+split: 20
+
+#### Network parameters
+
+depth:  3
+channels:  64
+latent_dims:  8
+pose_dims: 0
```

Comment:
Same here, could we set pose_dims default to at least 1 so that by default it runs our affinity-VAE code?